### PR TITLE
Dynamic Particles added

### PR DESCRIPTION
**Pull request by marizvi**
This PR is w.r.t #47 

Hello,
I have finally added the dynamic particles in background of our Welcome page, I hope you would like it.
If anything is missing or some extra changes required for the same then kindly let me know.
And if everything is fine then I would request you to kindly merge this small part of code as well with your main branch.

Thanks.

### CODE REVIEW

**Comment by marizvi** [comment]
hello,
I have added the particles to the bottom and also lowered down the opacity of those particles as well.
Now please if it is satisfactory then kindly merge my this small part of code, so that I could focus on other aspects of this website and give it more attractive look in future by raising more issues for the same..

Thank You.

**Comment by prathmeshkhandelwal1** [comment]
@marizvi I am not able to see the particles in the deploy preview, you can check yourself from the above deploy preview link. Make sure it is working and if you are still having issues with this. feel free to connect with us.

**Comment by prathmeshkhandelwal1** [comment]
> @marizvi I am not able to see the particles in the deploy preview, you can check yourself from the above deploy preview link. Make sure it is working and if you are still having issues with this. feel free to connect with us.

I am sure it was there in the last commit you made, see if the recent commit has some issues. maybe adding opacity is the reason(I am not sure tho )

**Comment by marizvi** [comment]
I don’t know what happened sir but it is working fine on my local server..
Still then i will see what might be the issue..

**Comment by marizvi** [comment]
yes you were right,
Opacity was creating the problem,
It has been solved now so you can take a look on it.

File: src/components/welcome/HomeComponent.css [hunk 1]
```diff
@@ -10,6 +10,17 @@
   font-size: 32px;
 }
 
+.particles{
```
Comment (prathmeshkhandelwal1) [inline src/components/welcome/HomeComponent.css:13]: no proper linting.

File: src/components/welcome/HomeComponent.js [hunk 2]
```diff
@@ -27,6 +28,9 @@ export default class HomeComponent extends Component {
 
         <Parallax.Layer offset={0} style={styles} speed={0.5} onClick={() => this.refs.parallax.scrollTo(1)}>
           <Header />
+          <div className = "particles2">
```
Comment (prathmeshkhandelwal1) [inline src/components/welcome/HomeComponent.js:32]: same here.

File: src/components/welcome/Header.js [hunk 3]
```diff
@@ -16,6 +17,10 @@ class Header extends Component {
     return (
       <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
         <div>
+        <div className = "particles">
```
Comment (prathmeshkhandelwal1) [inline src/components/welcome/Header.js:20]: No proper linting.

**Review by prathmeshkhandelwal1 — CHANGES_REQUESTED** [review]
Add linters in the codebase.

**Comment by prathmeshkhandelwal1** [comment]
Use prettier. @marizvi

**Comment by marizvi** [comment]
Thank You sir, for letting me know about "prettier", it was really amazing and time saving.
I have formatted the content with help of prettier.

Thanks.

**Comment by prathmeshkhandelwal1** [comment]
> Thank You sir, for letting me know about "prettier", it was really amazing and time saving. I have now formatted the content with help of prettier.
> 
> Thanks.

np ;)

**Comment by marizvi** [comment]
Thanku Sir.

**Pull request merged**